GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -11,28 +12,17 @@
 
 namespace GVFS.UnitTests.Prefetch
 {
-    [TestFixtureSource(TestRunners)]
+    [TestFixtureSource("ShouldTestSymLinkSupport")]
     public class DiffHelperTests
     {
-        public const string TestRunners = "Runners";
-
-        private static readonly bool[] SymLinkSupport = new bool[]
-        {
-            true,
-            false,
-        };
-
         public DiffHelperTests(bool symLinkSupport)
         {
             this.IncludeSymLinks = symLinkSupport;
         }
-
-        public static bool[] Runners
-        {
-            get { return SymLinkSupport; }
-        }
 
         public bool IncludeSymLinks { get; set; }
+
+        public static object[] ShouldTestSymLinkSupport = GVFS.Tests.Utilities.BooleanArrayForTestFixture;

```

Comment:
The NUnit documentation says that it 'may' exist in the test class. I'm not sure if that's a suggestion, or a requirement as the other items say 'must'. 

https://github.com/nunit/docs/wiki/TestFixtureSource-Attribute
```
It may be a field, property or method in the test class.
```

It works either way, but so did my bool[] code :)

Reply:
I don't think it says "may exist". I think it says "may be a field, property, or method" - as in you get to choose from that list. And "in the test class" refers to the current class if you don't specify any other, otherwise it refers to the class that you're passing to the attribute. So I think we're following the docs with my suggested syntax, and it matches Form 2 in the link you specified.